electron/launcher/src/_actions/user.actions.js

```diff
@@ -0,0 +1,130 @@
+import history from '../_helpers/history';
+
+const ipc = window.require('electron').ipcRenderer;
+
+// Action creators for user handling
+// API calls should go here?
+const login = (cred) => {
+  return (dispatch) => {
+    dispatch({type: 'LOGIN_START'});
+
+    ipc.send('http-login-credentials', cred);
+
+    ipc.on('http-login-credentials-success', (e, res) => {
+      dispatch({
+        type: 'LOGIN_SUCCESS',
```

Comment:
It may be better to reference states as variables than const values. Save this into a constants file. ie:
authentication_states.js
`LOGIN_SUCCESS = 'LOGIN_SUCCESS'`
Now you can reference them as:
authentication_states.LOGIN_SUCCESS instead of 'LOGIN_SUCCESS'

Reply:
Created new user.action.types.js file to hold these constants.